tests/unit/htdocs/migration/SuperglobalMigrationTest.php

```
@@ -0,0 +1,382 @@
+    /** @var array Backup of $_GET */
+    private array $backupGet;
+    /** @var array Backup of $_COOKIE */
+    private array $backupCookie;
+
+    protected function setUp(): void
+    {
+        $this->backupRequest = $_REQUEST;
+        $this->backupPost    = $_POST;
+        $this->backupGet     = $_GET;
+        $this->backupCookie  = $_COOKIE;
+    }
```

PHPMD (2.15.0)

[error] setUp accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)